index.html

```diff
@@ -1,19 +1,10 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>whatsup</title>
-    </head>
-    <body bgcolor="red">
-        
-        <h1>whatsup ganstas</h1>
-    </body>
-
-
-
-
-
-
-
-
-
-</html>
\ No newline at end of file
+   <head>
+      <title>whatsup</title>
+   </head>
+
+   <body bgcolor="red">
```

Comment:
please adjust the color to a more visually comfortable option.

Reply:
Fixed in 60ad9e19e2c782404c09a79433ca10437dca0486
Thank you for bringing this up!